subprojects/internal-integ-testing/src/main/groovy/org/gradle/test/fixtures/ivy/IvyDescriptorDependencyExclusion.groovy

```diff
@@ -17,6 +17,11 @@
 package org.gradle.test.fixtures.ivy
 
 class IvyDescriptorDependencyExclusion {
-    String org
-    String module
+      String org
```

Comment:
Can you fix the indentation here? Should be 4 spaces. Maybe you have tabs in there.